.github/workflows/build_and_test.yml

```diff
@@ -32,8 +32,9 @@ jobs:
         sudo apt update -qq
         sudo apt install -y default-jdk
     - uses: actions/checkout@v2
-    - uses: ros-tooling/setup-ros@v0.2
+    - uses: ros-tooling/setup-ros@v0.4
       with:
+        use-ros2-testing: true
```

Comment:
@jacobperron this would be equivalent to what the Rpr jobs use in other repos.
I think it's a good idea to test using the latest rolling packages.
But I can also check again without this option, there was a rolling sync recently so it should also be working with the packages in the main repo.

Reply:
I agree it is better to run CI against the testing repo. Thanks!